chrome-launcher.ts

```diff
@@ -118,15 +120,26 @@ export class Launcher {
     this.enableExtensions = defaults(this.opts.enableExtensions, false);
     this.connectionPollInterval = defaults(this.opts.connectionPollInterval, 500);
     this.maxConnectionRetries = defaults(this.opts.maxConnectionRetries, 50);
+
+    if (typeof this.opts.userDataDir === 'boolean') {
+      if (!this.opts.userDataDir) {
+        this.forceDefaultProfile = true;
```

Comment:
bikeshed: if it's a private property only, something like `useDefaultProfile` is another possible name

Reply:
OK